libs/core-ui/src/lib/Cohort/isAllDataCohort.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
what do you think about a more generic filename like allDataCohortUtils.ts?  Or defaultCohortUtils.ts?
I feel like more utils could eventually go into this file related to the "All Data" cohort eventually?

Reply:
I don't really care either way. But I can change that in the follow-up PR that I'm publishing later today. Marking as resolved but rest assured I'll do it there 😉